src/test/resources/test-project-response-code/build.gradle

```diff
@@ -0,0 +1,8 @@
+plugins {
+    id 'base'
+    id 'de.smartsquare.squit'
+}
+
+squit {
```

Comment:
This block is not needed in this test scenario, since we are not testing the database scripts.

Reply:
Ok